securedrop/source_app/info.py

```diff
@@ -1,20 +1,23 @@
 # -*- coding: utf-8 -*-
 import flask
-from flask import Blueprint, render_template, send_file, redirect, url_for
+from flask import Blueprint, render_template, send_file, redirect, url_for, flash
+from flask_babel import gettext
 import werkzeug
 
 from io import BytesIO  # noqa
 
 from encryption import EncryptionManager
 from sdconfig import SDConfig
+from source_app.utils import get_sourcev3_url
 
 
 def make_blueprint(config: SDConfig) -> Blueprint:
     view = Blueprint('info', __name__)
 
     @view.route('/tor2web-warning')
     def tor2web_warning() -> str:
-        return render_template("tor2web-warning.html")
+        flash(gettext("Tor2Web proxies do not protect your anonymity!"), "error")
+        return render_template("tor2web-warning.html", source_url=get_sourcev3_url())
```

Comment:
Probably this route should always return a 403?

Reply:
:heavy_check_mark: